src/script/lua_api/l_particles_local.cpp

```diff
@@ -0,0 +1,205 @@
+/*
```

Comment:
is it not possible to reuse the server's particles? Ie: by adding a member variable to indicate `is_local`